src/components/Trash.jsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import {Box,Button, Typography} from '@mui/material'
+import Paper from '@mui/material/Paper';
+import InputBase from '@mui/material/InputBase';
+import IconButton from '@mui/material/IconButton';
+import SearchIcon from '@mui/icons-material/Search';
+import MenuIcon from '@mui/icons-material/Menu';
+
+
+import { Note } from './customComponents/Note';
+import { Navbar } from './customComponents/Navbar';
+import { DeleteOutlineOutlined } from '@mui/icons-material';
+
+export const Trash = () => {
+    return <>
+    <div className="wrapper">
+    <aside>
+      <Box mb={4} sx={{ width: '100%', minWidth: 300, bgcolor: 'background.paper'}}>
+        <Navbar/>
+      </Box>
+      <Button variant="contained">Create New Note</Button>
+    </aside>
+        <main>
+             <div className="main-wrapper">
+                    <Paper component="form" sx={{ p: '2px 4px', display: 'flex', alignItems: 'center', width: 400 }}>
```

Comment:
suggestion: I feel inline styles can be avoided unless we need to use  a variable there

Reply:
Thanks. Have used the better styling method to overcome this.